packages/sanes-chrome-extension/src/logic/network/personafromconfig.ts

```diff
@@ -0,0 +1,26 @@
+import { getRuntimeConfiguration } from './runtimeconfiguration';
+import { Persona } from '../persona';
+import { getSignerAndProfile } from './signerandprofile';
+import { singleton } from '../../utils/singleton';
+
+const createPersonaFromConfig = async (): Promise<Persona> => {
```

Comment:
Unless I am wrong this is more or less what I did with PersonaFactory. This should be a kind of standard method to recreate persona each time the extension is open for:
A. Need to unlock it if the browser has just opened.
B. Just load it into memory for using it in the app (for example first usage should check it for navigating to  correct route).

Do not use it as a  singleton, using this approach, we need to have features to be able to clean it during tests sessions. Probably making Persona static, having a clear static method.

Reply:
A new Persona can now be created using Persona.create().

Now there is a `getGlobalPersona()` singleton as an intermediate step to bring this persona into the UI. If with #85 we can make sure that only one instance is created, great. Then `getGlobalPersona` can be removed.